be/src/olap/olap_meta.cpp

```diff
@@ -229,7 +229,7 @@ Status OlapMeta::put(rocksdb::WriteBatch* batch) {
 
     if (!s.ok()) {
         LOG(WARNING) << "rocks db put batch failed, reason:" << s.ToString();
```

Comment:
remove this log warning, constructor of Status will print all message.

Reply:
done